torch/_jit_internal.py

```diff
@@ -223,7 +223,8 @@ def can_compile_class(cls):
     names = cls.__dict__
     fns = [getattr(cls, name) for name in names if inspect.isroutine(getattr(cls, name, None))]
     has_code = [hasattr(fn, '__code__') for fn in fns]
-    return all(has_code)
+    fx_class = 'fx' in cls.__module__ if hasattr(cls, "__module__") else False
```

Comment:
Could you clarify a bit on why this is necessary? Thanks

Reply:
In `TestQuantizeFx.test_qat_and_script`, we try to script a `GraphModule` which has a `torch.fx.Graph` as an attribute. `Graph` is a regular object, not a `Module`, but we shouldn't try to script it.

I wonder if using the API I introduced in #45262 is a better solution here. We could ignore the attribute by adding `__ignored_attributes__` to `GraphModule` rather than doing this. What do you think?